Gemfile

```diff
@@ -5,7 +5,7 @@ gemspec
 gem 'claide', :git => 'https://github.com/CocoaPods/CLAide'
 
 # This is the version that ships with OS X 10.10, so be sure we test against it.
-gem 'json', '1.7.7'
+gem 'json', '1.8.3'
```

Comment:
I'm assuming you updated this because of OS X 10.11? I think we still support 10.10, so this shouldn't be done (also 10.11 seems to ship with 1.8.1?).